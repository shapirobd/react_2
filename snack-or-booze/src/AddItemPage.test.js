import React from "react";
import { MemoryRouter } from "react-router-dom";
import { render } from "@testing-library/react";
import App from "./App";

test("it should render without crashing", () => {
	render(
		<MemoryRouter initialEntries={["/add/drink"]}>
			<App />
		</MemoryRouter>
	);
});

test("snapshot test", () => {
	const { asFragment } = render(
		<MemoryRouter initialEntries={["/add/drink"]}>
			<App />
		</MemoryRouter>
	);
	expect(asFragment()).toMatchSnapshot();
});
