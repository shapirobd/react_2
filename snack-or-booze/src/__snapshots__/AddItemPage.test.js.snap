// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div>
      <nav
        class="navbar navbar-expand-md"
      >
        <a
          class="navbar-brand"
          href="/"
        >
          Snack or Booze
        </a>
        <ul
          class="ml-auto navbar-nav"
        >
          <li
            class="nav-item"
          >
            <a
              aria-current="page"
              class="active"
              href="/add"
            >
              Add
            </a>
            <a
              href="/snacks"
            >
              Snacks
            </a>
            <a
              href="/drinks"
            >
              Drinks
            </a>
          </li>
        </ul>
      </nav>
    </div>
    <main>
      <p>
        Loading â€¦
      </p>
    </main>
  </div>
</DocumentFragment>
`;
